plugin/src/main/groovy/com/novoda/staticanalysis/EvaluateToolViolationsTask.groovy

```diff
@@ -0,0 +1,20 @@
+package com.novoda.staticanalysis
+
+import org.gradle.api.DefaultTask
+import org.gradle.api.tasks.TaskAction
+
+class EvaluateToolViolationsTask extends DefaultTask {
```

Comment:
This task is the same as `EvaluateViolationsTask` but instead of holding a set of `Violations` it has a single `Violations` object, the one related to the specific tool